code/stable/nopcm/src/java/SWHS/Control.java

```diff
@@ -14,7 +14,7 @@ public static void main(String[] args) throws Exception {
         String inputfile = args[0];
         InputParameters inParams = new InputParameters();
         InputFormat.func_get_input(inputfile, inParams);
-        InputConstraints.input_constraints(inParams);
+        InputParameters.input_constraints(A_C, C_W, h_C, T_init, t_final, L, T_C, t_step, rho_W, D, T_W, E_W);
```

Comment:
and this I am quite sure can't compile.

Reply:
@JacquesCarette Right, NoPCM has never been able to compile. It's getting better but still quite broken.